pkg/webhooks/resource/handlers.go

```diff
@@ -145,9 +142,6 @@ func (h *handlers) Validate(ctx context.Context, logger logr.Logger, request *ad
 }
 
 func (h *handlers) Mutate(ctx context.Context, logger logr.Logger, request *admissionv1.AdmissionRequest, failurePolicy string, startTime time.Time) *admissionv1.AdmissionResponse {
-	if webhookutils.ExcludeKyvernoResources(request.Kind.Kind) {
```

Comment:
Should we not exclude the kyverno resources ?

Reply:
It was moved in the `WithFilter` middleware.

Reply:
got it , included the exclude resources in `WithFilter()`